src/components/SecurityShell/SecurityShell.js

```diff
@@ -0,0 +1,264 @@
+/**
+ * @file Shell.
+ * @copyright IBM Security 2020
+ */
+
+import ArrowLeft16 from '@carbon/icons-react/lib/arrow--left/16';
+import Close20 from '@carbon/icons-react/lib/close/20';
+
+import dataUri from 'data-uri.macro';
+import React, { Children, cloneElement, useRef, useState } from 'react';
+
+import { getComponentNamespace } from '../../globals/namespace';
+import theme from '../../globals/theme';
+
+import Button from '../Button';
+import HeaderNotification from '../SecurityHeader/HeaderNotification';
+import Icon from '../Icon';
+import IconButton from '../IconButton';
+import Link from '../Link';
+import ScrollGradient from '../ScrollGradient';
+
+const namespace = getComponentNamespace('header');
+const shellNamespace = getComponentNamespace('shell');
```

Comment:
IMO this could be called `legacyShellNamespace` to be crystal clear to anyone looking at this file in the future.